### ISSUE TITLE
Identifier must be [a-zA-Z0-9] is wrong

### ISSUE THREAD

**Issue by Cladis** [valhallasw/tools-contact#3]
I believe not all accounts actually match this pattern.

**Comment by valhallasw** [comment]
Yes, this is probably true, but given that this goes into an ldap search, I a) didn't really feel like figuring out how to escape stuff ;-), especially as I'm not sure how standardized that is, and b) as it's a substring match, it's generally easy enough to work around special characters (e.g. "Björn" could also be looked up with just 'bj'). However, if you know some docs on what escaping is necessary for ldap, I'd be happy to loosen up the regex.
